data/hai/unfettered.txt

```diff
@@ -0,0 +1,271 @@
+	"<, unfettered title>" ", space-monkey"
+		has "Unfettered Jump Drive 1: offered"
+	"<, unfettered title>" ", Captain"
+		has "unfettered: beaten the captain"
+
+
+phrase "cerberus hails"
+	word
+		"We do not accept your surrender!"
+		"You stand no chance!"
+		"Even if you take me down, my Masters will make quick work of you."
+		"Do not attempt to subdue me."
+		"I obey only the Masters."
+		"They hold my soul! Soon, they will hold yours as well."
+		"Die!"
+		"You have been allowed to live long enough. Now I come to cut the thread of your life."
+		"I may be a pawn, but at least I am on a greater path."
+		"You may never hope to achieve half of what I have done."
+		"My price is far above what you could possibly afford. Do not even attempt to bribe me."
+		"Do you hope to corrupt me? There is no price that is high enough."
+		"I guard the gates of Hell. You will not deter me from that."
+		"You think yourself special? You will die, just like the rest of you ignorant fools!"
+		"There can only be death for the likes of you."
+		"The Masters are superior to you."
+		"You may manage to kill me, but you will never break me."
+		"Run, you fool!"
+		"Killing you will allow me to ascend."
+		"When I will bring the husk of your ship to the Masters, I will be promoted."
+		"I look forward to killing you."
+		"Do you even know what you have gotten yourself into?"
+		"You will never go beyond that simple, mortal, shell!"
+		"Maybe in death you will find peace."
+		# the (reality destroyer/dimensional eraser) will be made safe
+		"Kashalam Eshar!"
+		# you will never be "prime"
+		"Uminor Magresh."
+		# your quest is void (futile)
+		"Porotar Elnam."
+
+
+phrase "unfettered challenger"
+	word
+		"The rules of the duel are clear: do not"
+		"Do not"
+		"Do not"
+		"You have asked for a time out, now do not"
+	word
+		" "
+		" attempt to "
+		" try to "
+		" presume to "
+		" think you can "
+	word
+		"attack me"
+		"shoot me"
+	word
+		" "
+	word
+		"until a day has passed"
+		"until you have changed systems or landed"
+		"until the time out is considered over"
+	word
+		", "
+	word
+		"or"
+		"else"
+		"if you do"
+	word
+		" "
+	word
+		"my brethren"
+		"fellow Unfettered"
+		"all true Hai"
+	word
+		" "
+	word
+		"will"
+		"shall be forced to"
+		"will not hesitate to"
+	word
+		"take action"
+		"shoot"
+		"defend me"
+		"do what it takes to maintain order"
+	word
+		"!"
+
+phrase "hostile unfettered challenger"
+	word
+		"Stop "
+		"Quit "
+	word
+		"shaming yourself"
+		"running away"
+		"overthinking"
+		"recharging your shields"
+		"hiding in fear"
+	word
+		" and come "
+		" and come try and "
+	word
+		"defeat "
+		"fight "
+	word
+		"me."
+	word
+		" Now is the time to reveal ourselves in the glory of battle"
+		" Show me what you got"
+		" Time for a fair fight"
+	word
+		"!"
+
+phrase "hostile unfettered challenger"
+	word
+		"Now is not the time for talk"
+		"Yes! Fight me"
+		"Our battle will be sang in legends for years to come"
```

Comment:
"sang" -> "sung of"

Reply:
```suggestion
		"Our battle will be sung of in legends for years to come"
```
You can use the suggestion button (with the +-) to do smth that looks like this

Reply:
Ah thanks, I wasn't able to find that before, hence the laborious hand-written notes.